android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -281,12 +283,23 @@ private void openUrlInBrowser(String url) {
     if (!URLUtil.isNetworkUrl(url)) {
       url = "http://" + url;
     }
+    // Route through the proxy server go link
+    url = createUrlProxyGoLink(url);
     // Open the browser and point it to the given url
     Intent intent = new Intent(Intent.ACTION_VIEW);
     intent.setData(Uri.parse(url));
     startActivity(intent);
   }
 
+  private String createUrlProxyGoLink(String url) {
```

Comment:
Agreed! I need to figure out a way to call for resources (e.g. R.string.proxy_go_link_base_url)  from a static method (otherwise Android complains). Thanks, I'll look into this.  =)